pkg/nsx/services/subnet/store.go

```diff
@@ -37,6 +38,16 @@ func subnetIndexFunc(obj interface{}) ([]string, error) {
 	}
 }
 
+// subnetTypeIndexFunc is used to filter out NSX Subnets which are tagged with subnetcr type.
+func subnetTypeIndexFunc(obj interface{}) ([]string, error) {
+	switch o := obj.(type) {
+	case model.VpcSubnet:
+		return filterTag(o.Tags, common.TagScopeSubnetCRType), nil
```

Comment:
I don't think we should add cr type in Tag, how about checking "nsx-op/subnetset_cr_uid" and "nsx-op/subnet_cr_uid"